qannotate/src/au/edu/qimr/qannotate/modes/TandemRepeatMode.java

```diff
@@ -318,76 +336,77 @@ else if ( (rep.patternLength < 6 &&  rep.patternLength > 0)
 	 */
 	BlockIndex makeIndexedBlock(final Set<Repeat> repeats){
 		
-		if(repeats == null || repeats.size() == 0) return null; 
+		if(repeats == null || repeats.size() == 0) {
+			return null; 
+		}
 		 
-			//S1: get unique block edge 
-			Set<Integer> starts = new HashSet<>(); //list will be very slow
-			Set<Integer> ends = new HashSet<>(); //list will be very slow
-			for(Repeat rep : repeats){				 
-				starts.add(rep.start);				
-				ends.add(rep.end);
-			}			
-			
-			//sort and unique the elements
-		   TreeSet<Integer> sortedStartEnd = new TreeSet<>();
-		    sortedStartEnd.addAll(starts);
-		    sortedStartEnd.addAll(ends);
-		    
-		    //S2:create block for each start end but the edge may require one base shift
-			Map<Integer, Block> blockIndex = new HashMap<>(); 
-			Iterator<Integer> it = sortedStartEnd.iterator(); 
-			int start = it.next();
-			int end = start; 
-			
-		    while(it.hasNext()){
-		    	
-			    	//do nothing if exists from starts disregards whether it is in ends or not
-			    	if(!starts.contains(start))
-			    		start += 1; 
-			    	
-			    	//trim if end is some repeat start
-			    	end = it.next();	
-			    	int end0 = end;  //store original boundary for next loop
-			    	 
-			    	if(starts.contains(end))		    		 
-			    		end -= 1; 
-			    	
-			    	if(start <= end)
-			    		blockIndex.put(start,  new Block(start, end));
-			    	
-			    	//next block must be one base forward
-			    	start = end0; 
-		    }
-		    
-		    //S3: filling repeat 
-		    for(Repeat rep : repeats){	
-				start = rep.start;	
-				//repeat must be greater than one base; 
-				//if end overlaop with next repeat start, chop the end
-				while(start < rep.end){
-					//all repeat start is never been chopped
-					Block block = blockIndex.get(start);
-					block.addRepeat(rep);
-					//next block must be one base forward
-					start = block.end+1;						
-				}
+		//S1: get unique block edge 
+		Set<Integer> starts = new HashSet<>(); //list will be very slow
+		Set<Integer> ends = new HashSet<>(); //list will be very slow
+		for(Repeat rep : repeats){				 
+			starts.add(rep.start);				
+			ends.add(rep.end);
+		}			
+		
+		//sort and unique the elements
+	   TreeSet<Integer> sortedStartEnd = new TreeSet<>();
```

Comment:
Since you're prettifying anyway, the indentation here and hereabouts still seems borked

Reply:
I've done some un-borking which has had the unfortunate effect of dismissing your review...